### DISCUSSION TITLE
Please add profiles

### DISCUSSION THREAD

**Discussion by Oneiromancer111** [Rem0o/FanControl.Releases#2074] [Ideas]
I love this software, considering how much functionality it has, im surprised it doesnt have profiles, like for example MSI afterburner.

As it is now you have to do 4 clicks when you want to change profile after opening the software  "1.three dots,  2, load configuration 3.  select configuration,  4, Load".

I used to do this with asus AI Suite 3,  i had 3 profiles with fixed speeds,  quiet mode,  medium gaming, and headphones on high speed.  For some reasons my fans acting strange now with AI suite, so i looked for alternative and found this software. 

Please add profile buttons! :)

**Comment by Rem0o** [comment]
Have you tried this?
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/cf0ed285-a9eb-47b7-98a5-c88597eb6e49)

**Comment by Oneiromancer111** [comment]
Oh thanks. That makes things easier,  To be honest I had just installed it, i didnt even get so far as to minimize it and discover that :)
I still would hope to see profiles when actually opening up the software.
